DirectEmployers/MyJobs
# [PD-3700] Add Combos to "apply_clicks" dimension

## What's this PR do?
Allow users to group apply clicks by job data: title, req_id and location (country, state, city)

## Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/27963853/30548888-59b02b98-9c61-11e7-9418-606253ac5eff.png)

## How should this be manually tested?
go to site traffic v2 report, and check out the `apply_clicks` report category, which is the first on the list. You will need Site Traffic V2 access locally, [here is how to set up](https://github.com/DirectEmployers/MyJobs/pull/3469)
